packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.scheduled-fargate-task.lit.ts

```diff
@@ -24,6 +25,7 @@ class EventStack extends cdk.Stack {
         cpu: 256,
         environment: { TRIGGER: 'CloudWatch Events' },
       },
+      eventRole: role,
```

Comment:
Passing in `undefined` as the role here is basically a no-op, and doesn't test anything. Can you pass in an actual role with the right permissions?